docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,52 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Image.html)
```

Comment:
I wrote this because other documents use noetic message type links, such as [control](https://github.com/autowarefoundation/autoware-documentation/blob/bb9cbd3cbca4c8f753b8f1323a91673459e1e2ff/docs/design/autoware-interfaces/components/control.md?plain=1#L11)

I will update all messages in sensing document soon.